### ISSUE TITLE
App built with versions 3.17.3, 3.18.0 crash on launch using Xcode 11. 

### ISSUE THREAD

**Issue by MindaugasJucius** [realm/realm-cocoa#6266]
!!! MANDATORY TO FILL OUT !!!

## Goals

Run the app without crashing
## Expected Results

For the app not to crash
## Actual Results

It crashes on launch:
```
* thread #1, queue = 'com.apple.main-thread', stop reason = signal SIGABRT
    frame #0: 0x000000010488b468 dyld`__abort_with_payload + 8
    frame #1: 0x000000010489228c dyld`abort_with_payload_wrapper_internal + 100
    frame #2: 0x00000001048922bc dyld`abort_with_payload + 12
    frame #3: 0x000000010488f6a8 dyld`dyld::halt(char const*) + 380
    frame #4: 0x00000001048445d0 dyld`dyld::fastBindLazySymbol(ImageLoader**, unsigned long) + 168
    frame #5: 0x000000019f0ccec0 libdyld.dylib`dyld_stub_binder + 60
    frame #6: 0x00000001045177f0 maxwise`type metadata accessor for NSDiffableDataSourceSnapshot<String, ExpenseCategory> at <compiler-generated>:0
    frame #7: 0x0000000104519c9c maxwise`type metadata completion function for ExpenseCategorySelectionViewController at <compiler-generated>:0
    frame #8: 0x00000001acaeeb10 libswiftCore.dylib`swift::MetadataCacheEntryBase<(anonymous namespace)::SingletonMetadataCacheEntry, int>::doInitialization(swift::ConcurrencyControl&, swift::MetadataCompletionQueueEntry*, swift::MetadataRequest) + 376
    frame #9: 0x00000001acae5658 libswiftCore.dylib`swift_getSingletonMetadata + 952
    frame #10: 0x000000010451730c maxwise`type metadata accessor for ExpenseCategorySelectionViewController at <compiler-generated>:0
    frame #11: 0x0000000104519c1c maxwise`ObjC metadata update function for ExpenseCategorySelectionViewController at <compiler-generated>:0
    frame #12: 0x000000019eff216c libobjc.A.dylib`realizeClassMaybeSwiftMaybeRelock(objc_class*, mutex_tt<false>&, bool) + 224
    frame #13: 0x000000019eff9090 libobjc.A.dylib`realizeAllClasses() + 112
    frame #14: 0x000000019effc210 libobjc.A.dylib`objc_copyClassList + 84
    frame #15: 0x000000010534c9c8 Realm`+[RLMSchema sharedSchema] + 332
    frame #16: 0x0000000105319ff4 Realm`+[RLMRealm realmWithConfiguration:error:] + 2124
    frame #17: 0x00000001067b66d4 RealmSwift`@nonobjc __C.RLMRealm.__allocating_init(configuration: __C.RLMRealmConfiguration) throws -> __C.RLMRealm + 148
    frame #18: 0x00000001067b6860 RealmSwift`RealmSwift.Realm.__allocating_init(configuration: RealmSwift.Realm.Configuration) throws -> RealmSwift.Realm + 200
  * frame #19: 0x00000001047683dc ExpenseKit`static Realm.groupRealm(self=RealmSwift.Realm) at Realm+Extensions.swift:15:29
```
## Steps to Reproduce

Try to run the app
## Code Sample

```
let fileURL = FileManager.default
            .containerURL(forSecurityApplicationGroupIdentifier: "id")
        guard let groupURL = fileURL else {
            fatalError("no groupURL")
        }
        
        let config = Realm.Configuration(fileURL: groupURL.appendingPathComponent("default.realm"))
        do {
            let realm = try Realm(configuration: config)
        } catch let error {
            print(error)
        }
```

It doesn't really matter if I try to instantiate Realm with a path to a group container or just `try Realm()` it still crashes.
## Version of Realm and Tooling

Realm framework version: 3.17.3/3.18.0

Realm Object Server version: ?

Xcode version: 11

iOS/OSX version: 13

Dependency manager + version: CocoaPods 1.5.3

**Comment by tgoyne** [comment]
A sample app which runs into problems would make it much easier to "Try to run the app".

**Comment by MindaugasJucius** [comment]
Fixed by updating from iOS 13 beta to iOS 13.1 👍
